components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -369,26 +368,30 @@ const ContributorProfileTab = ({
             <div className="flex justify-end">
               <DayRangePicker />
             </div>
-            <div className="flex flex-col justify-between gap-2 lg:flex-row md:gap-12 lg:gap-16">
+            <div className="grid grid-cols-2 2xl:grid-cols-4 justify-between gap-2 lg:flex-row md:gap-12 lg:gap-16">
```

Comment:
Fixed some styling while I was here as the addition to the OSCR didn't look good with the current layout.